### PR TITLE
Add v1 5 manifest header

### PR DESCRIPTION
**Pull request by zhuyingjiang**
add the 1.5 version manifest, for skylake and kabylake

### CODE REVIEW

**Comment by xiulipan** [comment]
@zhuyingjiang 
Please prefix the commit with rimage

> manifest: add 1.5 version manifest 
=>
rimage: add cavs 1.5 header 

And you can merge this two commit.

PS: how would you handle the currently used manifest. Will you change the name with 1.8?

**Comment by zhuyingjiang** [comment]
@lgirdwood 
1. please add me into this project, now I can't request a reviewer.
2. if separate version 1.5 and 1.8, do I need to change the old manifest to fw_image_manifest_v1_8?

**Comment by lgirdwood** [comment]
@zhuyingjiang yes, you will need to have separate structure names for manifest 1.5 and 1.8. Can you also fix check patch errors

**Comment by zhuyingjiang** [comment]
@lgirdwood have changed the old css header and manifest name to v1.8, and fixed the checkpatch

File: rimage/css.c [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 void ri_css_hdr_create(struct image *image)
 {
-	struct css_header *css = image->fw_image + MAN_CSS_HDR_OFFSET;
+	struct css_header_v1_8 *css = image->fw_image + MAN_CSS_HDR_OFFSET;
```
[the remark this answers is not in the document]
Comment (zhuyingjiang) [reply rimage/css.c]: OK, will rename this to v1.8 and add the 1.5 version

**Comment by zhuyingjiang** [comment]
@lgirdwood The API name changed to version 1.8, and 1.5 version header files still in this patch set. The 1.5 c code will send to you by patch first, and then send PR.

**Pull request merged**